src/raphael_backend_flask/templates/video_analysis.html

```diff
@@ -8,29 +8,100 @@
 </div>
 {% endif %}
 
-<div class="container mt-2 mb-6">
-  <div class="row justify-content-center sticky-top bg-white">
+<div class="container-fluid mt-2 mb-6">
+  <div class="row justify-content-center">
     <div class="col-lg-6 my-4 text-center">
       <div id="player"></div>
       <script async type="text/javascript" src="https://www.youtube.com/iframe_api"></script>
       <script>
       var player;
-      function onYouTubeIframeAPIReady() {
+      var currentTimeIntervalId;
```

Comment:
What do we think of using `let` instead of `var`?

Reply:
yes, good point!